include/basic_types.h

```diff
@@ -8,8 +8,15 @@
 
 #include <stdint.h>
 #include <arch/types.h>
+#include <assert.h>
 
-/* arch/types.h is supposed to define word_t and _seL4_word_fmt */
+/* arch/types.h defines wordRadix, wordBits is 2^wordRadix */
+#define wordBits     BIT(wordRadix)
```

Comment:
This will cause the some CI Preprocess complaints. But instead of using the `BIT(wordRadix)` macro we could also use the old `(1 << wordRadix)` definitions here. Seems to me `BIT()` is more intended for "real" bit masks, but here we just do some plain integer stuff.

Reply:
If the proofs can handle it without an update then it's likely fine.

Reply:
It looks like the proofs can't automatically handle the change between 1 and 1llu here.  Are you able to remove this commit as RISCV proofs use 1llu << wordRadix and the other proofs use 1<< wordRadix, and unifying them likely isn't high enough priority.